### DISCUSSION TITLE
private tangle API

### DISCUSSION THREAD

**Discussion by ziobongy** [gohornet/hornet#1588] [Q&A] [answered]
I have a problem with the APIs running a private tangle. I cloned the repository on my laptop, i went to the private_tangle folder, i bootstrapped the app and simply i ran the application by ./run.sh . The health endpoint return me a 200 http code but api returns me a 404 not found error.
I tried on 14266 port and 14265 port with addresses: localhost:14266/api/v1/info, localhost:14266/v1/info and so on, i tried every combination but nothing, i don't know how to make it works, can you help me please?

**Comment by muXxer** [comment, marked as answer]
If you cloned the repository, I guess you are on the `develop` branch.
There we are developing the upcoming `startdust` upgrade, which also changes the API.
Thats why the endpoint changed to `localhost:14265/api/core/v2/info`.

**Reply by ziobongy** [reply]
oh well, now it's working, thank you a lot
